### PR TITLE
Migrate core/workspace_comment_svg.js and core/workspace_comment_svg_render.js to goog.module syntax

### PR DESCRIPTION
**Pull request by moniika**
## The basics

- [x] I branched from `goog_module`
- [x] My pull request is against `goog_module`
- [x] My code follows the [style guide](
      https://developers.google.com/blockly/guides/modify/web/style-guide)
- [x] My code is presented in the form suggested in the [module
      conversion guide](https://github.com/google/blockly/issues/5026)
- [x] I have run `npm test` locally already.

### Resolves

Part of #5026

### Proposed Changes

Converts `core/workspace_comment_svg.js` and `core/workspace_comment_svg_render.js` to `goog.module` syntax.

### CODE REVIEW

File: core/workspace_comment_svg.js [hunk 1]
```diff
@@ -97,30 +102,60 @@ Blockly.WorkspaceCommentSvg = function(
...
-Blockly.WorkspaceCommentSvg.DEFAULT_SIZE = 100;
+WorkspaceCommentSvg.DEFAULT_SIZE = 100;
+
+/**
+ * Size of the resize icon.
+ * @type {number}
+ * @const
+ * @private
+ */
+WorkspaceCommentSvg.RESIZE_SIZE = 8;
```
Comment (rachel-fenichel) [inline core/workspace_comment_svg.js:133]: Since these are only used in this file, you can just declare them as `const RESIZE_SIZE = 8;` (and so on).

Comment (moniika) [reply core/workspace_comment_svg.js:133]: sgtm. I'll take a another pass and update those.

File: core/workspace_comment_svg.js [hunk 2]
```diff
@@ -46,6 +46,30 @@ goog.require('Blockly.Events.CommentMove');
 goog.require('Blockly.Events.Selected');
 
 
+/**
+ * Size of the resize icon.
+ * @type {number}
+ * @const
+ * @private
```
Comment (rachel-fenichel) [inline core/workspace_comment_svg.js:53]: No longer needs private annotation--it's private by virtue of not being exported.

**Review by rachel-fenichel — APPROVED** [review]
One nit on annotations, then lgtm.

**Pull request merged**